symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/test/java/com/symphony/bdk/bot/sdk/command/matcher/EscapedCharacterTest.java

```diff
@@ -0,0 +1,136 @@
+package com.symphony.bdk.bot.sdk.command.matcher;
+
+import org.junit.jupiter.api.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EscapedCharacterTest {
+
+  @Test
+  public void characterTest() {
+    EscapedCharacter matcher = EscapedCharacter.character('T');
+    assertEquals("T", matcher.regex());
+  }
+
+  @Test
+  public void characterWithReservedTest() {
+    String reservedCharacters = "+*?^$\\.[]{}()|/";
+    assertAll("All Reserved Character Escaped",
```

Comment:
I would prefer having only the `assertEquals`. Having `assertAll` + `assertEquals` is a bit redundant in my opinion.